docs/sphinx/source/whatsnew/v0.9.0.rst

```diff
@@ -111,6 +122,8 @@ Enhancements
   :func:`~pvlib.iotools.get_pvgis_hourly` for reading and retrieving hourly
   solar radiation data and PV power output from PVGIS. (:pull:`1186`,
   :issue:`849`)
+* Added `map_variables` option to :func:`~pvlib.iotools.get_pvgis_tmy` and
+  :func:`~pvlib.iotools.read_pvgis_tmy` (:issues:`1245`, :pull`1268`)
```

Comment:
Issue #1250 for this one, I think

Reply:
Nice! I had forgotten that I had created this issue actually.